spec/factories/symbols_factory.rb

```diff
@@ -0,0 +1,17 @@
+
+# frozen_string_literal: true
+
+FactoryGirl.define do
+  factory :symbol, class: OMSSymbol do
+    association :oms, factory: :oms
+    association :file_range
+    symbol_kind { %w(Op Pred Proposition Class Individual Name).sample }
```

Comment:
You're using `[..].map {}.sample` in other places. We should be consistent here

Reply:
These are not subclasses (hence, the differing column name) ;)